internal/metrics/metrics.go

```diff
@@ -60,7 +60,7 @@ func (c *ResourceMetricFetcher) fetchMetricsForResource(
 					Period: aws.Int32(60 * 60 * 24), // 1 day interval
 					Stat:   aws.String(statType),
 				},
-				Id: aws.String(strings.ToLower(metric)),
+				Id: aws.String(strings.ToLower(metric) + "_" + strings.ToLower(statType)),
```

Comment:
this is so if were getting both a sum and max of metric we don't  overwrite each other in final output.

so will have metrics in output like this now:
```
consumedreadcapacityunits_sum
consumedreadcapacityunits_maximum
```

vs before was just `consumedreadcapacityunits`